### ISSUE TITLE
Runtime error when changing checkSpelling:maxEditDistance

### ISSUE THREAD

**Issue by Devmacs** [justinwilaby/spellchecker-wasm#20]
The checkSpelling maxEditDistance is broken using the browser API, but I suspect it is also broken using the NodeJS API due to the similarity of the code.

Only the default works, any value greater than the default causes the runtime error (unreachable WASM error).

**Comment by Devmacs** [comment]
I can take this one.

**Comment by justinwilaby** [comment]
Looks like the fix is to apply the options to the `symspell` export and update the `defaultOptions`

```js
export const defaultOptions: SymSpellOptions = {
    dictionaryEditDistance: 2,
    countThreshold: 1
};
```

Then apply 
```js
symspell(options.dictionaryEditDistance, options.countThreshold);
```

The dictionaryEditDistance must be <= maxEditDistance otherwise the assertion in the `lookup` function fails

**Comment by justinwilaby** [comment]
Closing this as fixed with #21
